protocol/packages/swap/src/lib.rs

```diff
@@ -1,11 +1,37 @@
+#[cfg(feature = "osmosis")]
+pub use self::osmosis::*;
```

Comment:
let's specify the Impl only

Reply:
Not yet possible. Can be done only after refactoring the integration tests.